File: pset6/pset6/cash/cash.py
from cs50 import get_float


def main():
    change = 0.00   # def change variable in float
    while(change < 0.01):   # prompt user until good value float
        change = get_float("Change owed: ")

    cents = round(change * 100)
    print(f"Total cents: {cents}")  # print value of cents

    number_piece = 0
    while(cents > 0.00):    # give money while > 0.00
        while(cents >= 25):
            cents = cents - 25
            number_piece += 1
        while(cents >= 10):
            cents = cents - 10
            number_piece += 1
        while(cents >= 5):
            cents = cents - 5
            number_piece += 1
        while(cents >= 1):
            cents = cents - 1
            number_piece += 1

    print(f"Number of piece: {number_piece}")   # print final value number of piece


main()

File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>



int main(void)
{
    float i;
    do
    {
        i = get_float("Change owed: ");
    }
    while (i < 0.00);

    int cents = roundf(i * 100);
    printf("Total cents: %i\n", cents);

    int j = 0;
    do
    {
        while (cents >= 25)
        {
            cents = cents - 25;
            j++;
        }
        while (cents >= 10)
        {
            cents = cents - 10;
            j++;
        }
        while (cents >= 05)
        {
            cents = cents - 05;
            j++;
        }
        while (cents >= 01)
        {
            cents = cents - 01;
            j++;
        }
    }
    while (cents > 0.00);

    printf("%i\n", j);
}